packages/cross_file/test/x_file_io_test.dart

```diff
@@ -21,7 +21,7 @@ import 'package:cross_file/cross_file.dart';
 //
 // https://github.com/flutter/flutter/issues/20907
 
-final pathPrefix = './assets/';
+final pathPrefix = './test/assets/';
```

Comment:
Can you remove the comment above this pathPrefix? It's not true anymore (in master, at least)

Reply:
done